### ISSUE TITLE
repos of interest

### ISSUE THREAD

**Issue by cour10eygrace** [bcgov/SS_quantSppFeas#2]
1. whmacken/TreeSuitabilityPrediction-extrapolate to predict suitability-ordinal random forest (ordinal forest)
2. whmacken/SIBEC_Modelled - site index (growth rate curves) x site series  x species- (how tall is a tree at 50 years?)
3. bcgov/climr- climate model data inputs by site series 
4. whmacken/aSMR_X_rSMR - predicting actual soil moisture from relative soil moisture - how species respond to soil moisture
5. FLNRO-Smithers-Research/EdatopicGrid_Spp_Distribution -similar to models going to be setting up SDMs x site series

**Comment by cour10eygrace** [comment]
# 5 is rules for %area to feasibility? @kdaust

**Comment by cour10eygrace** [comment]
5 is rules for %area to feasibility? @kdaust

**Comment by kdaust** [comment]
> 5 is rules for %area to feasibility? @kdaust

Yep that's correct. Specifically, https://github.com/MoF-Skeena-Research/EdatopicGrid_Spp_Distribution/blob/master/2_0_EdatopicGrid_SpeciesDistribution.R
